projects/aws/upgrader/Help.mk

```diff
@@ -0,0 +1,64 @@
+
+
+
+########### DO NOT EDIT #############################
+# To update call: make add-generated-help-block
+# This is added to help document dynamic targets and support shell autocompletion
+
+
+##@ Image Targets
+local-images: ## Builds `upgrader/images/arm64` as oci tars for presumbit validation
+images: ## Pushes `upgrader/images/push` to IMAGE_REPO
+upgrader/images/arm64: ## Builds/pushes `upgrader/images/arm64`
+upgrader/images/push: ## Builds/pushes `upgrader/images/push`
```

Comment:
```suggestion
upgrader/images/amd64: ## Builds/pushes `upgrader/images/amd64`
```
Right?

Reply:
again same as above, its auto-generated. might not be the right command. but this needs to be fixed at the generation side. can create an issue if that helps!